packages/app/src/systems/Transaction/machines/transactionsMachine.tsx

```diff
@@ -0,0 +1,76 @@
+import type { Transaction } from 'fuels';
+import { createMachine } from 'xstate';
+
+type MachineContext = {
+  walletAddress: string;
+  txs: Transaction[];
+  error?: string;
+};
+
+type MachineServices = {
+  fetchTxs: {
+    data: Transaction[];
+  };
+};
+
+type MachineEvents = {
+  type: 'reload';
+};
+
+export const transactionsMachine = createMachine(
+  {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+    tsTypes: {} as import('./transactionsMachine.typegen').Typegen0,
+    schema: {
+      context: {} as MachineContext,
+      services: {} as MachineServices,
+      events: {} as MachineEvents,
+    },
+    id: 'FETCH ADDRESS TRANSACTIONS',
```

Comment:
try to follow the pattern we already have in other machines here for `states` names and machine `id`